File: composite/java/Cliente.java
package composite.java;

public class Cliente {
    public static void main(String[] args) {
        File file1 = new File("documento1.txt", 100);
        File file2 = new File("documento2.txt", 200);

        Directory subDirectory = new Directory("Subdirectorio");
        subDirectory.add(file1);

        Directory mainDirectory = new Directory("Directorio Principal");
        mainDirectory.add(subDirectory);
        mainDirectory.add(file2);

        System.out.println("Tamaño total: " + mainDirectory.getSize()); // 300
    }
}

File: composite/python/Cliente.py
from Leaf import File
from Composite import Directory

file1 = File("documento1.txt", 100)
file2 = File("documento2.txt", 200)

sub_directory = Directory("Subdirectorio")
sub_directory.add(file1)

main_directory = Directory("Directorio Principal")
main_directory.add(sub_directory)
main_directory.add(file2)

print(f"Tamaño total: {main_directory.get_size()}")  # 300